### PR TITLE
Removed all uses of "using".

### PR DESCRIPTION
**Pull request by inkstink**
This is the first time I attempted to commit anything to stockfish.  I went through the code and removed all uses of "using", I fixed some spacing issues that this change caused.

### CODE REVIEW

**Comment by mcostalba** [comment]
@kvanvel  Why?

**Comment by lucasart** [comment]
`using` dumps the entire content of a namespace into the global namespace. this is an environmental crime, which can cause name conflicts, and negates the point of using namespaces. plus use of using is largely inconsistent across the code. sometimes, it's `std::string`, sometimes, it's `string`, etc. and without using code is more readable, because it is context free, and you know where things come from.

File: src/misc.cpp [hunk 1]
```diff
@@ -64,10 +62,18 @@ struct Tie: public streambuf { // MSVC requires split streambuf for cin and cout
...
-  Logger() : in(cin.rdbuf(), file.rdbuf()), out(cout.rdbuf(), file.rdbuf()) {}
+  Logger()
+    :
+    in(std::cin.rdbuf(),
+       file.rdbuf()
+    ),
+    out(std::cout.rdbuf(),
+	file.rdbuf()
+    )
+    {}
```
Comment (lucasart) [inline src/misc.cpp]: I don't think this kind of pedantic spacing improves readability. If you don't like compact one-liners, how about ?
```
Logger() :
    in(std::cin.rdbuf(), file.rdbuf()),
    out(std::cout.rdbuf(), file.rdbuf())
{}
```

**Comment by snicolet** [comment]
Personally, I think that the code is much more readable before the patch than after.

**Comment by mcostalba** [comment]
@inkstink thanks for taking teh time to write this patch and open the PR.

I hope you will continue and push other PR. Unfortunately I am going to close this one becuase of no use, but please don't let this to discourage yourself.

@lucasart of course it is inconsistent, in the headers you don't want to place 'using' directives, you may place them mainly in *.cpp files and only where they improve readibility: 'using' directive is not something you apply 100% of cases or never.

**Pull request closed without merging**